plugins/janus_videoroom.c

```diff
@@ -506,6 +506,7 @@ typedef struct janus_videoroom_publisher {
 	gint64 fir_latest;	/* Time of latest sent FIR (to avoid flooding) */
 	gint fir_seq;		/* FIR sequence number */
 	gboolean recording_active;	/* Whether this publisher has to be recorded or not */
+	gboolean recording_pause;
```

Comment:
This is nitpicking, I know, but `recording_paused` is a better name for this. Can you also add a comment on the side, as we do for the other properties?